screencastr.thor

```diff
@@ -16,49 +16,59 @@ class Screencastr < Thor
   # 6. Upload video to S3
     # a. Will need a naming scheme to match the location properly
 
+  class_option :framerate, aliases: "-f", desc: "Specify a framerate for OUT_FILE, (eg. -f 30)", type: :numeric
+  class_option :resolution, aliases: "-r", desc: "Specify a resolution for OUT_FILE, (eg. -r 1920x1080)", type: :string
+
   desc "add_bumpers IN_FILE OUT_FILE", "Add bumpers to video"
-  method_option :framerate, aliases: "-f", desc: "Specify a framerate for OUT_FILE, (eg. -f 30)", type: :numeric
-  method_option :resolution, aliases: "-r", desc: "Specify a resolution for OUT_FILE, (eg. -r 1920x1080)"
   def add_bumpers(in_file, out_file)
     ProcessHelpers.resolution_require_value(options[:resolution])
-    video = FFMPEG::Movie.new(in_file)
+    ext = File.extname(out_file)[1..-1]
+
+    `ffmpeg -f lavfi -i anullsrc=channel_layout=stereo:sample_rate=44100 \
+    -loop 1 -i assets/1080-GA-Splash.png -f #{ext} -t 5 -r #{options[:framerate] || 30} -pix_fmt yuv420p \
+    -vf scale=#{options[:resolution] || "1920x1080"} -map 0:a -map 1:v bumper.#{ext}`
```

Comment:
This is where I'm converting the static image into a video. Note that I've added an empty audio track. This is for later, when we concatenate this video to the screencast.

Reply:
This is nice, I knew that having an audio track would simplify the concat, but I didn't know how to create it.